queries/inko/highlights.scm

```diff
@@ -0,0 +1,150 @@
+; Brackets and operators
+[
+  "("
+  ")"
+  "["
+  "]"
+  "{"
+  "}"
+] @punctuation.bracket
+
+[
+  "+"
+  "-"
+  "/"
+  "*"
+  "**"
+  "%"
+  "<"
+  ">"
+  "<="
+  ">="
+  "<<"
+  ">>"
+  ">>>"
+  "&"
+  "|"
+  "^"
+  "=="
+  "!="
+] @operator
+
+; Keywords
+[
+  "and"
+  "as"
+  "case"
+  "class"
+  "enum"
+  "extern"
+  "fn"
+  "for"
+  "if"
+  "impl"
+  "import"
+  "let"
+  "mut"
+  "ref"
+  "trait"
+  "uni"
+  "loop"
+  "match"
+  "else"
+  "move"
+  "or"
+  "recover"
+  "return"
+  "throw"
+  "try"
+  "while"
+  (modifier)
+  (visibility)
+] @keyword
```

Comment:
Some of these should be more specific (e.g. `@keyword.conditional`). Check CONTRIBUTING.md.